README.md

```diff
@@ -32,11 +32,14 @@ Also, if you want to request a particular package, you may suggest it in an issu
 
 If your device running Android OS with version 7 and higher, it is highly recommended to check whether your Termux installation uses our new repository with packages compiled specially for higher Android API levels (24+).
 
-Execute command `termux-info` or look into file `$PREFIX/etc/apt/sources.list`. There should be a line containing following URL:
-```
-https://dl.bintray.com/termux/termux-packages-24/
+Execute any one of these commands:
+```bash
+$ termux-info | grep -o 'https://dl.bintray.com/termux/termux-packages-[(0-9)]*' | cut -d- -f3'
+
+$ cat $PREFIX/etc/apt/sources.list | grep -o 'https://dl.bintray.com/termux/termux-packages-[(0-9)]*' | cut -d- -f3'
 ```
```

Comment:
Same as above.
Why not to prompt user to download some shell script then ?

Reply:
> Why not to prompt user to download some shell script then ?
Downloading and running is more cumbersome then copy-pasting